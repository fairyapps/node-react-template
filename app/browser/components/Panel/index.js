import styled from "styled-components";

export const Panel = styled.div`
  background-color: #fff;
  border-radius: 3px;
  padding: 20px;
  box-shadow: 0px 1px 1px rgba(0, 0, 0, 0.1);
`;

export default Panel;
