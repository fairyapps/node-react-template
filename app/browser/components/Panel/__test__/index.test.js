import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import Panel from "../index";

test("Panel renders correct snapshot", () => {
  const component = renderer.create(<Panel>sometext</Panel>);
  let tree = component.toJSON();
  expect(tree).toMatchStyledComponentsSnapshot();
});
