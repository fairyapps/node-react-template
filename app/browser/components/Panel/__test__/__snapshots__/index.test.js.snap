// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Panel renders correct snapshot 1`] = `
.fxCqnE {
  background-color: #fff;
  border-radius: 3px;
  padding: 20px;
  box-shadow: 0px 1px 1px rgba(0, 0, 0, 0.1);
}

<div
  className="sc-bdVaJa fxCqnE"
>
  sometext
</div>
`;
